benchmark/scala/src/main/scala/com/microsoft/tpch/ConvertTpchDatasetToParquet.scala

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+// The code is modified based on Wentao's code. Wentao's homepage: https://www.microsoft.com/en-us/research/people/wentwu/
+
+package mypackage
+
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{SaveMode, SparkSession}
+
+object ConvertTpchCsvToParquetApp {
+
```

Comment:
remove empty line. You set up the scala-fmt as specified here: https://github.com/dotnet/spark/blob/master/docs/coding-guidelines/scala-coding-style.md, right?

Reply:
Oh, I just set up the scale-fmt, I was using IntelliJ as the formatter in the Editor Settings. Thanks.